index.html

```diff
@@ -1,75 +1,106 @@
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+      </section>
+
+      <section class="middle-content">
+        <h2>Why Did It Have To Be Boxes...</h2>
 
-        <section class="bottom-content">
+        <div class="boxes">
+          <div class="box">Box 1</div>
+          <div class="box">Box 2</div>
+          <div class="box">Box 3</div>
+          <div class="box">Box 4</div>
+          <div class="box">Box 5</div>
+          <div class="box">Box 6</div>
+          <div class="box">Box 7</div>
+          <div class="box">Box 8</div>
+          <div class="box">Box 9</div>
+          <div class="box">Box 10</div>
+        </div>
+      </section>
 
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
+      <section class="bottom-content">
+        <div class="text-container">
+          <h2>That's</h2>
+          <p>
+            Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed
+            tellus vestibulum ultrices quis in nunc.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>No</h2>
+          <p>
+            Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed
+            tellus vestibulum ultrices quis in nunc.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>Moon</h2>
+          <p>
+            Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed
+            tellus vestibulum ultrices quis in nunc.
+          </p>
+        </div>
+      </section>
 
-        </section>
-        
-        <footer>
-            <nav>
-                <a href="#">Home</a>
-                <a href="#">About</a>
-                <a href="#">Products</a>
-                <a href="#">Blog</a>
-                <a href="#">Contact</a>
-            </nav>
-        </footer>
-    </div><!-- container -->        
-</body>
-</html>
\ No newline at end of file
+      <footer>
+        <nav>
+          <a href="#">Home</a>
+          <a href="#">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+      </footer>
+    </div>
+    <!-- container -->
+  </body>
+</html>
```

Comment:
Good job! Missing your about page. How comfortable do you feel using the git workflow?